### ISSUE TITLE
Small problem when runing pgcenter with AmazonRDS.

### ISSUE THREAD

**Issue by kalamba** [lesovsky/pgcenter#35]
Hello! I don't now it's an issue or smth else, but this can be useful.

When I try to run pgcenter with AmazonRDS and touch "x" for switching to pg_stat_statements, I catched error: "ERROR: permission denied for relation pg_authid".

Trouble is that the "pg_authid" system table is not accessible even for the RDS master user because it stores information (passwords) that needs to be protected to maintain service security and stability.
But subset of "pg_authid" information is available in "pg_roles" but the latter table doesn't contain user passwords.

So I had to change some files from you source code.
```
sed -i 's/pg_authid/pg_roles/g' ./share/doc/pgcenter.1
sed -i 's/pg_authid/pg_roles/g' ./src/include/qstats.h
sed -i 's/pg_authid/pg_roles/g' ./src/include/queries.h
```
After make && make install and pg_stat_statements working fine.

**Comment by lesovsky** [comment]
Looks good, thanks!

**Comment by lesovsky** [comment]
fixed with 6e334727
